server/odc-migrate/src/main/resources/migrate/web/V_4_2_4_16__add_stateful_route.sql

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
42416 is occupied by this pr #1782

and why should we do this in a version script?

Reply:
Move script from data.sql to web version script, cause stateful should only be opened in web mode.